config.js

```diff
@@ -0,0 +1,29 @@
+const { clientId, clientSecret, sessionSecret } = require('./credentials.json');
+
+module.exports = {
+  apiBase: 'https://photoslibrary.googleapis.com/v1',
+  oAuth: {
+    callbackURL: 'http://localhost:8080/auth/callback',
```

Comment:
let's also use an env here, or we could have separate config files for diff envs, but it looks like this is the only var that changes across envs